components/badge/badge.css

```diff
@@ -0,0 +1,93 @@
+.badge-wrapper {
+  position: relative;
+  width: fit-content;
+}
+
+.badge {
+  position: absolute;
+  border-radius: 50%;
```

Comment:
You can make a utility class for border radius-50% as it will be used a lot of time ,otherwise everything looks great.

Reply:
Okay, I will change it. thank you